Microsoft.DotNet.Interactive/LanguageService/LanguageServiceExtensions.cs

```diff
@@ -28,5 +30,39 @@ public static T ToLspObject<T>(this JObject obj)
         {
             return obj.ToObject<T>(_jsonSerializer);
         }
+
+        private const string DataUriPrefix = "data:";
+
+        public static string DecodeDocumentFromDataUri(this string dataUri)
+        {
+            // from https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs, basic format is
+            // data:[<mime_type>][;base64],<data>
+            if (!dataUri.StartsWith(DataUriPrefix))
+            {
+                throw new FormatException($"Expected string to start with '{DataUriPrefix}'");
```

Comment:
This should be an `ArgumentException`.

Reply:
Resolved when I renamed the method to `Try...`.